app.go

```diff
@@ -62,7 +62,9 @@ type APNConfig struct {
 
 type FirebaseConfig struct {
 	Enabled              bool   `json:"enabled"`
+	ServerKey            string `json:"server_key"`
 	NotificationTemplate string `json:"notification_template"`
+	DataTemplate         string `json:"data_template"`
```

Comment:
good to add `omitempty`

Reply:
normally we should separate request and response structs since they have different fields.